postinstall.tsconfig.json

```diff
@@ -1,6 +1,6 @@
 // WORKAROUND https://github.com/angular/angular/issues/18810
-// This file is required to run ngc on angular and other libraries,
-// to write files like node_modules/@angular/core/core.ngsummary.json
+// This file is required to run ngc on 3rd party libraries such as @ngrx,
+// to write files like node_modules/@ngrx/store/store.ngsummary.json
```

Comment:
but it still has the fatal flaw that it doesn't work when that library has any `@Component`s right? maybe we should note that to be transparent

Reply:
yes.. there is that flaw. I'll can update that comment